src/tools/web/elektrad/src/app.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express'
+import bodyParser from 'body-parser'
+
+import initRoutes from './routes'
+
+const PORT = 1234
```

Comment:
This also seems to be still an issue (especially because 1234 is not really an uncommon port).